app/src/main/java/com/example/itbook/data/source/local/dao/BookDaoImp.kt

```diff
@@ -12,7 +12,7 @@ class BookDaoImp(booksDatabase: BooksDatabase) : BookDao {
 
     override fun getBook(isbn13: String): Book? {
```

Comment:
```suggestion
    override fun getBook(id: String): Book? {
```
Nếu đặt tên hợp lý thì sẽ dễ hiểu như trên